backend/authentication/views.py

```diff
@@ -59,3 +62,50 @@ def echo(self, request: Request) -> Response:
             return Response(token_serializer.validated_data)
 
         raise AuthenticationFailed(token_serializer.errors)
+
+
+data = {
+    "id": "1234",
+    "username": "test",
+    "email": "test@test",
+    "first_name": "test",
+    "last_name": "test",
+}

```

Comment:
I'd prefer to see this in `ypovoli.settings.py`.
Keep all the configuration together.